js/apps/system/_admin/aardvark/APP/react/src/views/collections/addCollection/AddCollectionForm.tsx

```diff
@@ -0,0 +1,170 @@
+import { Divider, Grid, Stack, Text } from "@chakra-ui/react";
+import React from "react";
+import { InputControl } from "../../../components/form/InputControl";
+import { useCollectionsContext } from "../CollectionsContext";
+import { useField } from "formik";
+import { SwitchControl } from "../../../components/form/SwitchControl";
+import { useFetchDatabase } from "../../databases/useFetchDatabase";
+import { getCurrentDB } from "../../../utils/arangoClient";
+import { SelectControl } from "../../../components/form/SelectControl";
+import { CreatableMultiSelectControl } from "../../../components/form/CreatableMultiSelectControl";
+
+export const AddCollectionForm = () => {
+  const { database, error } = useFetchDatabase(getCurrentDB().name);
+  const { collections, isFormDisabled: isDisabled } = useCollectionsContext();
+  const [distributeShardsLike] = useField<string>("distributeShardsLike");
+  const [isSatellite] = useField<boolean>("isSatellite");
+  if (error) {
+    window.App.navigate("#collections", { trigger: true });
+    window.arangoHelper.arangoError(
+      "Failure",
+      `Got unexpected server response: ${error.errorNum}: ${error.message}`
+    );
+  }
+  if (!database) return null;
+  const isOneShard = database.isOneShard || window.frontendConfig.forceOneShard;
+  return (
+    <Grid templateColumns={"1fr"} gap="6">
+      <Stack>
+        <Text>Basic</Text>
+        <Divider borderColor="gray.400" />
+        <Grid templateColumns={"1fr 1fr"} columnGap="4" alignItems="start">
+          <InputControl isDisabled={isDisabled} name="name" label="Name" />
+          <SelectControl
+            isDisabled={isDisabled}
+            name={"type"}
+            label="Type"
+            selectProps={{
+              options: [
+                { label: "Document", value: "document" },
+                { label: "Edge", value: "edge" }
+              ]
+            }}
+            tooltip={
+              "Use the Document type to store json documents with unique _key attributes. Can be used as nodes in a graph. <br> Use the Edge type to store documents with special edge attributes (_to, _from), which represent relations. Can be used as edges in a graph."
+            }
+          />
+        </Grid>
+        <Text>Configuration</Text>
+        <Divider borderColor="gray.400" />
+        <Grid
+          templateColumns={"1fr 1fr"}
+          rowGap="5"
+          columnGap="3"
+          marginY="4"
+          maxWidth="full"
+        >
+          {window.App.isCluster && (
+            <>
+              <InputControl
+                isDisabled={
+                  isDisabled || isOneShard || distributeShardsLike.value !== ""
+                }
+                name={"numberOfShards"}
```

Comment:
I think this field is only supposed to be seen when we are in the coodinator DB, do check